src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_ExcelCompat.txt

```diff
@@ -2084,3 +2084,283 @@ Error({Kind:ErrorKind.InvalidArgument})
+"19:23:15.35"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00")
+"00:00:00.00"
+
+>> Text(Time(14, 32, 25, 678), "hh:mm:ss.00;hh/mm/ss")
+"14:32:25.67"
+
+>> Text(Time(-5, 23, 15, 356), "hh:mm:ss.00;hh/mm/ss")
+"19/23/15"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00;hh/mm/ss")
+"00:00:00.00"
+
+>> Text(Time(14, 32, 25, 678), "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"14:32:25.67"
+
+>> Text(Time(-5, 23, 15, 356), "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"19/23/15"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"00 00 0"
+
+>> Text(Time(14, 32, 25, 678), "\T\:hh:mm:ss.00;\Z\:hh/mm/ss")
+"T:14:32:25.67"
+
+>> Text(Time(-5, 23, 15, 356), """T:""hh:mm:ss.00;""Z:""hh/mm/ss")
+"Z:19/23/15"
+
+>> Text(Time(0, 0, 0, 0), """T:""hh:mm:ss.00;""Z:""hh/mm/ss")
+"T:00:00:00.00"
+
+>> Text(Time(14, 32, 25, 678), "\T\:hh:mm:ss.00;\Z\:hh/mm/ss;\K\:hh mm ss")
+"T:14:32:25.67"
+
+>> Text(Time(-5, 23, 15, 356), "\T\:hh:mm:ss.00;\Z\:hh/mm/ss;\K\:hh mm ss")
+"\Z\:19/23/15"
+
+>> Text(Time(0, 0, 0, 0), "\T\:hh:mm:ss.00;\Z\:hh/mm/ss;\K\:hh mm ss")
+"\K\:00 00 0"
+
+>> Text(Time(14, 32, 25, 678), "hh:mm:ss.00\;hh/mm/ss")
+"14:32:25.67\;14/32/25"
+
+>> Text(Time(-5, 23, 15, 356), "hh:mm:ss.00\;hh/mm/ss")
+"19:23:15.35\;19/23/15"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00\;hh/mm/ss")
+"00:00:00.00\;00/00/00"
+
+>> Text(Time(14, 32, 25, 678), "hh:mm:ss.00\;hh/mm/ss\;hh mm ss")
+"14:32:25.67\;14/32/25\;14 32 25"
+
+>> Text(Time(-5, 23, 15, 356), "hh:mm:ss.00\;hh/mm/ss\;hh mm ss")
+"19:23:15.35\;19/23/15\;19 23 15"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00\;hh/mm/ss\;hh mm ss")
+"00:00:00.00\;00/00/00\;00 00 00"
+
+>> Text(Time(14, 32, 25, 678), "hh:mm:ss.00"";""hh/mm/ss"";""hh mm ss")
+"14:32:25.67;14/32/25;14 32 25"
+
+>> Text(Time(-5, 23, 15, 356), "hh:mm:ss.00"";""hh/mm/ss"";""hh mm ss")
+"19:23:15.35;19/23/15;19 23 15"
+
+>> Text(Time(0, 0, 0, 0), "hh:mm:ss.00"";""hh/mm/ss"";""hh mm ss")
+"00:00:00.00;00/00/00;00 00 00"
+
+>> Text(25, "yyyy-mm-dd;mm/dd/yyyy")
+"1900-01-24"
+
+>> Text(-25, "yyyy-mm-dd;mm/dd/yyyy")
+"12/05/1899"
+
+>> Text(0, "yyyy-mm-dd;mm/dd/yyyy")
+"1899-12-30"
+
+>> Text(25, "yyyy-mm-dd;mm/dd/yyyy;mmmm dd yyyy")
+"1900-01-24"
+
+>> Text(-25, "yyyy-mm-dd;mm/dd/yyyy;mmmm dd yyyy")
+"12/05/1899"
+
+>> Text(0, "yyyy-mm-dd;mm/dd/yyyy;mmmm dd yyyy")
+"December 30 1899"
+
+>> Text(25, "yyyy-mm-dd\;mm/dd/yyyy")
+"1900-01-24;01/24/1900"
+
+>> Text(-25, "yyyy-mm-dd\;mm/dd/yyyy")
+"1899-12-05;12/05/1899"
+
+>> Text(0, "yyyy-mm-dd\;mm/dd/yyyy")
+"1899-12-30;12/30/1899"
+
+>> Text(25, "yyyy-mm-dd\;mm/dd/yyyy\;mmmm dd yyyy")
+"1900-01-24;01/24/1900;January 24 1900"
+
+>> Text(-25, "yyyy-mm-dd\;mm/dd/yyyy\;mmmm dd yyyy")
+"1899-12-05;12/05/1899;December 05 1899"
+
+>> Text(0, "yyyy-mm-dd\;mm/dd/yyyy\;mmmm dd yyyy")
+"1899-12-30;12/30/1899;December 30 1899"
+
+>> Text(1246.234, "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"05:36:57.60"
+
+>> Text(-34.56, "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"10/33/36"
+
+>> Text(0, "hh:mm:ss.00;hh/mm/ss;hh mm ss")
+"00 00 0"
```

Comment:
This should be "00 00 00" since we have 2 's', right?

Reply:
updated.